content/hardware/02.hero/boards/uno-r4-wifi/tutorials/cloud-setup/cloud-setup.md

```diff
@@ -49,28 +49,20 @@ When the installation is finished you will see a short loading screen and once y
 ![Configure Device](./assets/configureDevice.png)
 
 
-Choose a name for your board and press next. The next parts need a little bit more attention. Because the UNO R4 WiFi doesn't have a crypto chip it needs a **secret key** that works as an identifier, guaranteeing a safe connection. That secret key is shown to you in the next step and you can either download a pdf containing all the information you need to save or copy and paste the secret key and the device ID someplace where you won't lose it.
+Choose a name for your board and press next.
 
-***Note: If you don't save the secret key you will not be able to continue with the next step so make sure you document and save it! In case you lose it, start over and you will receive a new key.***
+![Name your Arduino Board](./assets/naming.png)
 
-![Secret Key](./assets/secretKey.png)
-
-To use the UNO R4 WiFi with the IoT Cloud you may need to update the firmware of the ESP32 module. Luckily we have simplified the process of flashing new firmware to the module. 
-
-You will be prompted with a popup when it's time. Follow the instructions depending on what operating system you are using. 
-
-![Update Board](./assets/updateDevice.png)

```

Comment:
@Hannes7eicher remember to remove this image as well! The linter picks up on it being "not used" 👍🏼